### ISSUE TITLE
khmer v2.1 - next release after 2.0

### ISSUE THREAD

**Issue by ctb** [dib-lab/khmer#1393]
Notes:

With #1380 we will be adding --quiet to many scripts, so we need to increment minor version number.

**Comment by standage** [comment]
We don't have a specific checklist or timeline slated for this release, do we?

**Comment by ctb** [comment]
Nope!

We should revisit the '2.0+' issues and retag them as '2.1' as necessary.

**Comment by betatim** [comment]
We are getting close, should we make a release candidate?

**Comment by betatim** [comment]
Just went through the checklist for everything but uploading to the test instance of pypi. If we merge #1688 I will make a rc1 with a tag on github etc.

If one of the people who have the account name and password for pypi could share that would be cool.

**Comment by ctb** [comment]
what's your pypi account?
On Tue, May 02, 2017 at 08:52:11AM -0700, Tim Head wrote:
> Just went through the checklist for everything but uploading to the test instance of pypi. If we merge #1688 I will make a rc1 with a tag on github etc.
> 
> If one of the people who have the account name and password for pypi could share that would be cool.
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/dib-lab/khmer/issues/1393#issuecomment-298676911
-- 
C. Titus Brown, ctbrown@ucdavis.edu

**Comment by betatim** [comment]
betatim

**Comment by ctb** [comment]
you now have maintainer access

**Comment by betatim** [comment]
I will make v2.1rc1 tomorrow morning (my time) unless I hear otherwise.

**Comment by betatim** [comment]
You can install khmer v2.1rc1 with:
```
pip install -i https://testpypi.python.org/pypi --pre --no-clean khmer
```

The pypi page needs a bit of massaging as it doesn't render the README's markdown: https://testpypi.python.org/pypi/khmer/2.1rc1

**Comment by standage** [comment]
Awesome. I installed successfully in a fresh virtualenv. (I had to install screed separately, I assume due to using the PyPI test database).

Regarding rendering the README, there has been an open request on the PyPI issue tracker to support Markdown rendering since 2014. I don't see that happening anytime soon. But take a look at https://github.com/pypa/pypi-legacy/issues/148#issuecomment-227757822 for a potential solution. It's a bit of an ugly hack, but if we do the conditional import of `pypandoc` then we only have to make it a requirement for developers cutting a release, which I think is reasonable.

**Comment by betatim** [comment]
-> #1695 for readme conversion PR

**Comment by ctb** [comment]
For next RC, I think we should include fixes for #1572 (see #1680, which should probably be reopened).

Also what about the guzzle theme #1630? we should get that fixed and merged.

**Comment by standage** [comment]
👍
